self-tests.yml

```diff
@@ -200,6 +200,17 @@ functions:
             make mod-tidy;
             [[ $? -eq 0 ]] && break;
           done
+  get-swaggo:
+    command: shell.exec
+    params:
+      shell: bash
+      working_dir: evergreen
+      script: |
```

Comment:
Same thing as below (sorry I read this PR bottom-up for some reason...) - can we turn this script into a make target or a script under `scripts`? The make target makes it convenient for users to reproduce this in their own local environment.

Reply:
Sounds good!